src/scss/components/_buttonhalloffame.scss

```
@@ -0,0 +1,51 @@
+.buttonHallOfFame {
+  display: grid;
+  grid-template-columns: 133px 1fr;
+  justify-content: space-around;
```

Mała zmiana zgodnie z konwencją na którą uwagę zwrócił mi Patryk:
```suggestion
  display: grid;
  justify-content: space-around;
  grid-template-columns: 133px 1fr;

```